docs/central-users.rst

```diff
@@ -19,67 +19,94 @@ Central features Role-based User permissioning. In the current release of Centra
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Comment                          | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| View & Download                  | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| OData Access                     | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entity Lists**                                                                             |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create                           | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| List All                         | x             | x               | x              |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Add Properties                   | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Update Settings                  | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| **Project Entities**                                                                                 |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create from Submission           | x             | x               |                | x              |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| Create Directly                  | x             | x               |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| View in Linked Forms             | x             | x               | x              | x              |
++----------------------------------+---------------+-----------------+----------------+----------------+
+| | View & Download Anywhere,      | x             | x               | x              |                |
+| | Not Just in Linked Forms       |               |                 |                |                |
++----------------------------------+---------------+-----------------+----------------+----------------+
```

Comment:
You can see that this text is slightly misaligned in the screenshot in the description. It's like it's missing left padding. I tried another RST syntax for multiple lines: instead of the second `|`, a blank line between the multiple lines. But the first line ended up wrapping awkwardly.

```rst
+----------------------------------+---------------+-----------------+----------------+----------------+
| View & Download Anywhere,        | x             | x               | x              |                |
|                                  |               |                 |                |                |
| Not Just in Linked Forms         |               |                 |                |                |
+----------------------------------+---------------+-----------------+----------------+----------------+
```

<img width="824" src="https://github.com/getodk/docs/assets/5970131/ba0e2e8e-a1eb-4da3-8f14-b3154c2fdb7b">